Transition from Jekyll to Statik

mattjoseph89 opened the pull request:
Our goal is to transition the HTML layouts, partials and Sass from Jekyll over to Statik. We will be operating Statik within the parent folder from Jekyll until all of the data necessary has been harvested. At that point we will delete all files relating to Jekyll out, and allow Statik to take over as the sole file structure within this project.

sumeetjain commented on app/controllers/genres_controller.rb:
```diff
@@ -0,0 +1,10 @@
+class GenresController < ApplicationController
+  def index
```
Remove unused methods (genrescontroller#index)

sumeetjain replied:
Has approx 6 statements (TooManyStatements)

sumeetjain commented on app/controllers/artists_controller.rb:
```diff
@@ -1,10 +1,10 @@
 class ArtistsController < ApplicationController
   def show
-    @artist = Artist.find_by_id(params[:id])
+    @artist        = Artist.find_by_id(params[:id])
```
Calls params[:id] 3 times (DuplicateMethodCall)

sumeetjain commented on app/controllers/genres_controller.rb:
```diff
@@ -0,0 +1,10 @@
+class GenresController < ApplicationController
```
Has no descriptive comment (IrresponsibleModule)

sumeetjain replied:
Has at least 6 instance variables (TooManyInstanceVariables)

sumeetjain commented on app/models/release.rb:
```diff
@@ -21,7 +8,7 @@ def self.info_by_decade
 
     while decade >= 1860 do
-      decade_releases[decade] = self.where("release_date >= ? AND release_date <= ?", decade, (decade + 10))
+      decade_releases[decade] = self.where("release_date >= ? AND release_date < ?", decade, (decade + 10))
```
Line is too long. [107/80]

sumeetjain commented on app/models/album.rb:
```diff
@@ -2,10 +2,24 @@ class Album < ActiveRecord::Base
   end
+  # Retrieve tracks associated with a given album and return hash with track number keys and track title values
+  # NOT YET WORKING
+  def self.tracks
```
Has approx 6 statements (TooManyStatements)

sumeetjain replied:
Use empty lines between method definitions.

sumeetjain commented on app/models/album.rb:
```diff
@@ -2,10 +2,24 @@ class Album < ActiveRecord::Base
   def album_image
     AlbumImage.find_by_album_id(self.id)
   end
+  # Retrieve tracks associated with a given album and return hash with track number keys and track title values
```
Line is too long. [111/80]

sumeetjain commented on app/models/album.rb:
```diff
@@ -2,10 +2,24 @@ class Album < ActiveRecord::Base
+    track_number = 1
 
+    while track_number <= album_tracks.count
+     tracklist[track_number] = album_tracks.title
```
Use 2 (not 1) spaces for indentation.

mattjoseph89 committed:
Transition from Jekyll to Statik

Our goal is to transition the HTML layouts, partials and Sass from Jekyll over to Statik. We will be operating Statik within the parent folder from Jekyll until all of the data necessary has been harvested. At that point we will delete all files relating to Jekyll out, and allow Statik to take over as the sole file structure within this project.

mattjoseph89 committed:
statik folder in place and gitignore setup for public folder

mattjoseph89 committed:
artist/album relation setup

sumeetjain committed:
Link artist to artist page
